src/ChannelServer/Database/Account.cs

```diff
@@ -87,6 +88,26 @@ public void AddChatMacro(ChatMacro macro)
 		/// <returns></returns>
 		public ChatMacro[] GetChatMacros()
 		{
+			// Initialize with the default set of macros if none exist.
+			if (_chatMacros.Count() == 0)
+			{
+				var macroData = ChannelServer.Instance.Data.ChatMacroDb
+					.Where(x => x.Id <= 10) // maximum number of chat macros allowed.
+					.OrderBy(x => x.Id);
+
+				if (macroData == null)
```

Comment:
This will never be null, because you're guaranteed to get an enumerator from the above call.